src/lib/caching/structures/guilds/GuildMember.ts

```diff
@@ -94,6 +94,19 @@ export class GuildMember extends Structure {
 		return this.nick ?? this.user?.username ?? null;
 	}
 
+	/**
+	 * The displayed color of the member
```

Comment:
colour is spelt with a U, also missing full stop
```suggestion
	 * The displayed colour of the member.
```